ts/store/reducers/entities/messages/messagesById.ts

```diff
@@ -72,7 +72,7 @@ const reducer = (
         ...state,
         [id]: {
           ...prevState,
-          message: pot.noneError(action.payload.error)
+          message: pot.noneError(action.payload.id)
```

Comment:
having the state being indexed with respect to the id of the message whose loading fails, here we could pass `action.payload.error` to get details on the kind of error occurred.
```suggestion
          message: pot.noneError(action.payload.error)
```

Reply:
updated within 7fd3edd1095bee221db15492d60b85eb6a2cfcb6